cernopendata/modules/fixtures/data/docs/lhcb-getting-started/lhcb-getting-started.md

```diff
@@ -1,84 +1,8 @@
-*   Look at the fitted mass distribution. You can split it into three regions: the signal region and two background-only "sidebands": one above the signal (the upper sideband) and one below the signal (the lower sideband). A Gaussian distribution contains 99.7% of its events within three standard deviations of the mean, so this "three σ" region around the mean is usually the definition of the signal region.
-*   Use the slider labelled Sig range to define the beginning and end of the signal region. All events not falling into the signal region will be said to fall into the background region.
-*   You can now use the definitions of the signal and background regions in the mass variable to determine the signal and background distributions in other variables. Click on the button labelled `Apply cuts and plot variables`. You will see the signal (blue) and background (red) distributions for the other three variables plotted next to the mass distribution. You should discuss the exercise with an instructor at this point.
-
-**Exercise 2** : measuring the D<sup>0</sup> lifetime The object of this exercise is to use the signal sample which you obtained in the previous step to measure the lifetime of the D<sup>0</sup> particle. This is the same quantity as the half-life of a radioactive particle: the D<sup>0</sup> decays according to an exponential distribution, and if this exponential is fitted to a distribution of the D<sup>0</sup> decay times, the slope of this exponential is the lifetime of the D<sup>0</sup>.
-
-*   Fit the lifetime of the D<sup>0</sup>.
-*   Compare the slope of this exponential to the D<sup>0</sup> lifetime given by the Particle Data Group. Talk to an instructor about how well these agree with each other.
-*   In addition to statistical uncertainties, measurements can suffer from systematic uncertainties, caused by a miscalibrated apparatus or an incorrect modelling of the backgrounds. One basic technique for estimating these is to repeat the measurement while changing the criteria used to select signal events. If the result changes significantly when changing the criteria, we know that there is something wrong!
-*   Repeat your fit for the lifetime of the D<sup>0</sup> while varying the maximum allowed D<sup>0</sup> impact parameter. The allowed values range from -4:0 to 1.5 in the original fit. Move this upper value from 1.5 to -2.0 in steps of 0.25, and refit the D<sup>0</sup> lifetime at each point, saving the results as you go along.
-*   Plot the histogram showing the fitted value of the D<sup>0</sup> lifetime as a function of the upper cut on the impact parameter. Discuss the shape, and what it tells you about the D<sup>0</sup> lifetime, with an instructor.
-*   What other sources of systematic uncertainty might we need to consider when making a lifetime measurement?
-
+In 2022, LHCb has released some of the data via CERN OpenData Portal, making it available to the public.
```

Comment:
The new Getting started seems to be just a skeleton; people wouldn't be able to proceed much further from there, and there are no links, etc.  Do you plan to enrich it?

Please note also that there is https://opendata.cern.ch/docs/about-lhcb which may be perhaps better place for some of the notes you have wrote here?

Reply:
Yes, this is a skeleton, the "getting started" before was actually a page with instructions on LHCb masterclass exercise. Getting started page would be enriched with some instructions on how to use open data (and a short description of what data and docs are there). So all the content from here moves to "masterclass" pages.